libs/extensions/angular/image-banner/src/image-banner.component.ts

```diff
@@ -36,6 +36,11 @@ export class ImageBannerComponent {
    */
   @Input() externalLink: string | undefined;
 
+  /**
+   * The text of the dismiss button. Will be used as title for sighted users and aria-label for accessibility.
+   */
+  @Input() dismissButtonText: string | undefined;
```

Comment:
How about, instead of adding this as an input, we use the existing translation-service for this, like we do in e.g. the [modal close button](https://github.com/kirbydesign/designsystem/blob/develop/libs/designsystem/modal/src/modal-wrapper/modal-wrapper.component.html#L23)? 
We could concatenate it with the title for the banner, to improve the labelling.